src/docs/install.md

```diff
@@ -59,6 +59,24 @@ NB: The formatting of the plugins list is important; all lines below the `plugin
 indented by at least one white space to form logical continuation lines. This is standard for python
 ini files, see [[Options|pants('src/docs:options')]].
 
+Pants' runtime Python interpreter
+---------------------------------
+Pants may be ran with Python 2.7, 3.6, or 3.7. The `./pants` script defaults to using Python 2.7. It's recommeded, though, that you pin the version you'd like to use.
+
+To do this, add an entry like so to `pants.ini`, choosing between `2.7`, `3.6`, or `3.7`.
```

Comment:
It would be nice to improve the symmetry between the `pants_version` pinning section and this section (or even merge them?)... unfortunately, I think that doing that would require adding an equivalent of `./pants -v` that would render the version it is running under... or some black magic in the `setup` script to dump that info and exit..?